apps/asap-server/test/fixtures/users.fixtures.ts

```diff
@@ -361,6 +364,7 @@ export const userResponse: UserResponse = {
   questions: ['Question 1', 'Question 2'],
   avatarUrl: `${config.baseUrl}/api/assets/${config.appName}/squidex-asset-id`,
   role: 'Grantee',
+  labs: [],
```

Comment:
thats not really a great fixture if you ask me - you dont need to implement anything for this test to pass, we could just return an empty array. It's ok to test that case too - as in - the empty result, but I would first make sure that we get actual labs here, that goes for both route testing and controller testing (both of which use this fixture)